### PR TITLE
#185528797 Fix disabled account shouldn't be able to login

### PR DESCRIPTION
**Pull request by jcharles921**
## What does this PR do?
This PR fix a bug on user  blocked  that were able to login and perform actions
## How should this be manually tested?

- Clone the repository using `git clone` [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-bn)

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm run dev`

## What are the relevant pivotal tracker/Trello stories?

[#185528797](https://www.pivotaltracker.com/story/show/185528797)

## Screenshots 

image

### CODE REVIEW

**Review by nishimweprince — APPROVED** [review]
Excellent fix!!

**Review by bergad37 — APPROVED** [review]

File: src/routes/auth.js
Comment (irakevine) [inline src/routes/auth.js:1]: It's great that you removed the invalid route and installed the correct one

**Review by irakevine — APPROVED** [review]
Great Work!

**Review by ReneNtabana — APPROVED** [review]
Great work

**Review by mken1205 — APPROVED** [review]

**Pull request merged**